packages/gatsby-theme-docs/README.md

```diff
@@ -152,6 +154,67 @@ The available JSX components are:
 
 > When using JSX components, it's recommended to leave a **blank line** between the element tags and the actual content. This allows the content to be parsed as markdown, so you can use markdown syntax within the custom component tags.
 
+## Using Theme with Add-Ons
+
+A theme add-on is a Gatsby Theme that exposes React components to be injected into the MDX provider of the core theme.
+
+Usually it's possible for a child theme to use components shadowing (see [Theme overrides](#theme-overrides)). However, with multiple themes, the shadowed components are loaded from the last theme child. To solve this problem, a commercetools-docs Gatsby Theme can be used as an add-on.
+
+When using add-on themes, a proxy export file will be created in the websites `src/@commercetools-docs/gatsby-theme-docs/overrides` folder to leverage Gatsby's component shadowing (see [Theme overrides](#theme-overrides)), including all the exported components from the add-on packages.
+
+**We recommend to use the `configureThemeWithAddOns` function to configure the websites's `gatsby-config.js`**
+
+```js
+const {
+  configureThemeWithAddOns,
+} = require('@commercetools-docs/gatsby-theme-docs/configure-theme');
+
+module.exports = {
+  plugins: [
+    ...configureThemeWithAddOns({
+      // Pass the normal theme options
+      websiteKey: 'my-website-key',
+      // Define the add-on plugins
+      addOnPlugins: [
+        '@commercetools-docs/gatsby-theme-foo',
+        {
+          resolve: '@commercetools-docs/gatsby-theme-bar',
+          options: {
+            // ...
+          },
+        },
+      ],
+    }),
+  ],
+};
+```
+
+Alternatively you should pass the Gatsby theme add-ons as normal plugins and additionally reference the package names in the `themeAddOns` option of the commercetools core theme.
```

Comment:
(we spoke about this today, just for documentation here): I suggest to not advertise this third option, but to be clear that add ons have to be configured through the configuration function  if they are used.